django_airavata/wagtailapps/base/migrations/0020_auto_20190416_1644.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please remove this file.

Reply:
Hello Marcus,

I applied the requested changes above and from Travis CI. Please let me know if I need to close my previous PR.

Thanks,
Keerthi